code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -81,6 +81,12 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	//PV = nRT
 
 	///joules per kelvin
+
+/datum/gas_mixture/proc/clear(list/tocheck)
+	var/list/cached_gases = gases
+	for(var/id in (tocheck || cached_gases))
+		cached_gases -= id
```

Comment:
This is functionally the same as this:
```suggestion
	gases,Cut()
```
Are you appending more behavior here?

Reply:
so i don't need to add the clear() proc i just can do Cut(), also i added it before but right now i don't even need it, i'll remove it